iron-autogrow-textarea.html

```diff
@@ -167,6 +168,15 @@
       },
 
       /**
+       * The value for this input, same as `bindValue`
+       */
+      value: {
+        notify: true,
+        type: String,
+        computed: '_computeValue(bindValue)'
```

Comment:
I think in a future PR we should maybe rename `bind-value` to `value`? It is `bind-value` because this used to be a `<textarea is="iron-textarea">` which is no longer the case.